mage/src/client/pages/ResultPage.jsx

```diff
@@ -139,12 +139,11 @@ export const ResultPage = () => {
           (path) =>
             path !== ".env.server" &&
             path !== ".env.client" &&
-            path !== "src/client/vite-env.d.ts" &&
-            path !== "src/client/tsconfig.json" &&
-            path !== "src/server/tsconfig.json" &&
-            path !== "src/shared/tsconfig.json" &&
+            path !== "src/vite-env.d.ts" &&
+            path !== "tsconfig.json" &&
             path !== ".gitignore" &&
-            path !== "src/.waspignore" &&
+            path !== ".waspignore" &&
+            path !== "public/.gitkeep" &&
```

Comment:
What about package-lock.json? We want to show that?

Reply:
We are not running the apps, so no package-lock.json is generated for now.